src/index.html

```diff
@@ -1,13 +1,55 @@
+  <!-- #region Head-->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion-->
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img 
+      src="images/imac.jpeg" 
+      alt="imac"
+      class="product-card__img"
+    >
+      <div class="product-card__info">
+
+        <h5 class="product-card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h5>
+
+        <span class="product-card__product-code">Product code: 195434</span>
+
+        <div class="product-card__review">
+          <div class="stars stars--0">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star stars__star--off"></div>
```

Comment:
```suggestion
            <div class="stars__star"></div>
```

Reply:
no need to off star directly, it should work dynamically depends on modifier for `stars` block